app/src/main/java/com/hamdan/forzenbook/viewmodels/LoginViewModel.kt

```diff
@@ -274,3 +276,25 @@ fun LoginViewModel.LoginState.toLoginEntrys(): LoginEntrys =
         email = this.email,
         code = this.code,
     )
+
+fun LoginViewModel.LoginState.toLoginComposeState(): LoginComposeState =
+    LoginComposeState(
```

Comment:
States shouldn't have `Compose` like this in their name. The state of the screen does not depend on compose or legacy. The state is the state and can be shared between ui types